src/components/CreateForm/styledtextinput.js

```diff
@@ -14,7 +14,6 @@ const StyledTextInput = styled.fieldset`
 		padding: 1em;
 		border: 0;
 		border-radius: 0.4em;
-		text-align: center;
 	}
 	input[type='number'] {
 		width: 30%;
```

Comment:
I would like to recommend you not to nest your styles.  You can use props and the Styled Components `css` helper instead.  If the refactor takes too much time, feel free to put this item on the list for next week after you make the video.